zerver/webhooks/sentry/doc.md

```diff
@@ -20,6 +20,11 @@ you like. If you want notifications for issues and not just events,
 you can scroll down to **Webhooks** on the same page and check the
 box that says **issues**. Make sure that you set up the permissions
 so that the integration will visible to the right people.
+**NOTE**: Zulip also supports configuring this as a webhook in Sentry &mdash;
```

Comment:
Maybe put a newline before this so that it is its own paragraph?

Reply:
Making it a new paragraph seems to create a new `<ol>` in the output HTML. I'm not sure how to prevent that from happening. :see_no_evil:

Reply:
Fixed. Needed to indent the new paragraph.